tools/changelog/changelog.go

```diff
@@ -85,6 +85,7 @@ func main() {
 	from := os.Args[1]
 	to := os.Args[2]
 
+	// #nosec
```

Comment:
There is no reason to analyze tools/ for security problems. It's not a part of production code.

Reply:
Well, it is part of our codebase nonetheless.

Reply:
I'm almost positive we can remove this tool from our codebase.

Reply:
I got thrilled now. How can we be 100% positive?

Reply:
We can remove it and wait until someone notices it :)